.github/workflows/lint.yml

```diff
@@ -31,3 +31,4 @@ jobs:
       - uses: hadolint/hadolint-action@v3.1.0
         with:
           dockerfile: Containerfile
+          ignore: DL3018
```

Comment:
better to put this ignore in the dockerfile so that locally we dont have an error also